### PR TITLE
Pod name should contain cluster name.

### PR DESCRIPTION
**Pull request by paulc4**

### CODE REVIEW

**Review by vegichan — APPROVED** [review]
quick and easy! Thanks paul

File: stable/admin/templates/NOTES.txt [hunk 1]
```diff
@@ -6,4 +6,4 @@ NuoDB can be accessed via port 48004 on the following DNS name from within your
 
 To display your NuoDB cluster state run:
 
-  $ kubectl exec admin-{{ .Values.admin.domain }}-0 -- nuocmd show domain
+  $ kubectl exec admin-{{ .Values.admin.domain }}-{{ .Values.cloud.cluster.name }}-0 -- nuocmd show domain
```
Comment (sivanov-nuodb) [inline stable/admin/templates/NOTES.txt:9]: To be more accurate I suggest that we use the following template:
```
$ kubectl exec {{ include "admin.fullname" . }}-0 -- nuocmd show domain
```

**Review by sivanov-nuodb — COMMENTED** [review]
Thanks so much, Paul for fixing this! I added one suggestion.
I think that we have the same problem in the `database` chart notes.

File: stable/admin/templates/NOTES.txt (hunk 1 above)
Comment (adriansuarez) [reply stable/admin/templates/NOTES.txt:9]: +1

**Review by sivanov-nuodb — APPROVED** [review]

**Review by adriansuarez — APPROVED** [review]

**Pull request merged**